conda/models/__init__.py

```diff
@@ -14,3 +14,21 @@
 - ``conda.models``
 
 """
+from ..common.compat import text_type
+
+
+def translate_feature_str(val):
```

Comment:
Per the email thread, 'nomkl' would be great to translate to a more "positive" result, like "openblas" or "accelerate"